include/exercise_functions.php

```diff
@@ -0,0 +1,472 @@
+<?php
+
+function addExercise(){
+
+  $exerciseTypeId = getExerciseTypeId();
+
+  $exerciseName = $_REQUEST["exerciseName"];
+  $session = $_REQUEST["session"];
+  $exerciseTarget = $_REQUEST["target"];
+  $userId = $_SESSION['userId'];
+
+  db_Query("
+    INSERT INTO exercise(exerciseName, typeId, session, target, userId)
+    VALUES(:exerciseName, :typeId, :session, :target, :userId)
+  ",
+  
+  [
+    'exerciseName' => $exerciseName,
+    'typeId' => $exerciseTypeId,
+    'session' => $session,
+    'target' => $exerciseTarget,
+    'userId' => $userId
+  ]
+
+  
+  );
+}
+
+function getExerciseTypes(){
+
+  $exerciseTypeData = db_Query("
+    SELECT *
+    FROM exercise_type
+  ")->fetchAll();
+
+  return $exerciseTypeData;
+
+}
+
+function getExerciseTypeId(){
+
+  $exerciseTypeName = $_REQUEST["exerciseType"];
```

Comment:
same here about `excerciseType` being a parameter. I'm gonna stop making that comment but probably most of the time its better to take in parameters for functions instead of trusting the state of the app. That way these functions always work as you would expect them to instead of factors outside of the functions control changing their outcome (this can be described as a `function programming paradigm`)